packages/experimental/Button/src/CompoundButton/SPEC.md

```diff
@@ -0,0 +1,81 @@
+# Compound Button
+
+## Background
+
+The `CompoundButton` is a `Button` that is designed to show an extra line of text.
+
+## Sample Code
+
+Basic examples:
```

Comment:
Is it possible to put an image here of what one of these examples looks like? I know it's pretty self-explanatory, but I always think seeing it visually along with the code snippet helps :)

Reply:
Sure, I'll add some visuals.

My hope is that in the long run the website which will house the documentation will allow us to have code snippets with visual examples of the button. Documentation in the repo is a short-term thing.